tests/test_adjoint_jacobian.py

```diff
@@ -125,12 +125,11 @@ def test_not_expval(self, dev):
             qml.RX(0.1, wires=0)
             qml.state()
 
-        if device_name == "lightning.kokkos":
-            message = "Adjoint differentiation does not support State measurements."
+        if device_name in ("lightning.qubit", "lightning.kokkos"):
+            message = "Adjoint differentiation method does not support measurement StateMP."
```

Comment:
Is this the general message for new API devices? If yes, could you please re-write it in terms of ``_new_API``?

Reply:
Ok, I will change it

Reply:
Done! :rocket: